ludwig/datasets/__init__.py

```diff
@@ -180,6 +272,9 @@ def get_datasets_output_features(
         for competition in competition_datasets:
             del ordered_configs[competition]
 
+    for hf_dataset in hugging_face_datasets:
```

Comment:
It looks like there's only one config whose name is "hugging_face". Could we instead do:

```python
del ordered_configs["hugging_face"]
```